tests/integration/support/createNamespace.js

```diff
@@ -7,16 +7,11 @@ Cypress.Commands.add('createNamespace', namespaceName => {
 
   cy.openCreate();
 
-  cy.contains('Simple').click();
-
-  cy.get('ui5-dialog')
+  cy.get('.create-form')
     .find('[aria-label="Namespace name"]:visible')
     .find('input')
     .type(namespaceName, { force: true })
     .click();
 
-  cy.get('ui5-dialog')
-    .contains('ui5-button', 'Create')
-    .should('be.visible')
-    .click();
+  cy.createResource();
```

Comment:
I just noticed that this custom command is redundant, because we can also just write
`cy.saveChanges('Create');`
so please change this here and also remove the command `createResource` from create-edit-support.js